build-tools/xaprepare/xaprepare/Application/Scenario.cs

```diff
@@ -58,6 +58,7 @@ protected virtual void AddStartSteps (Context context)
 		{
 			// These are steps that have to be executed by all the scenarios
 			Steps.Add (new Step_DownloadNuGet ());
+			Steps.Add (new Step_InstallPackagesLocally ());
```

Comment:
Does this need to be in the *default* `Scenario.AddStartSteps()` implementation?

Wouldn't `Scenario_Required` make more sense, or `Scenario_AndroidToolchain`?

Reply:
When building locally for a daily development cycle you don't usually run scenarios other than the default one, I think this is the right place to put this.

Reply:
This is a good point, I think it may make more sense to move this into the default (`Standard`) scenario, rather than the base class for all scenarios.